udata_front/views/reuse.py

```diff
@@ -1,8 +1,8 @@
-from flask import abort, request, url_for
-from werkzeug.contrib.atom import AtomFeed
+from flask import abort, request, url_for, make_response
+from feedgenerator.django.utils.feedgenerator import Atom1Feed
 
 from udata_front.views.base import SearchView, DetailView
-from udata.i18n import I18nBlueprint, lazy_gettext as _
+from udata.i18n import I18nBlueprint, gettext as _
```

Comment:
Modified due to import error in `speaklater` underlying dependency. Inspired from https://github.com/opendatateam/udata/pull/1381/files